src/rdkafka_conf.c

```diff
@@ -1013,6 +1013,38 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "the client's OAUTHBEARER token.",
           _UNSUPPORTED_OAUTHBEARER
         },
+        { _RK_GLOBAL, "sasl.oauthbearer.method", _RK_C_STR,
+          _RK(sasl.oauthbearer.method),
+          "Enable the functionality of the OAuth/OIDC login callback handler.",
```

Comment:
The callback is an internal detail which is not exposed to the user, so it should not be mentioned here, and the help text needs to be a bit more clear what this property can be set to and what it does, and how it relates to the other new properties, so that the user knows when to set it and what other properties are needed.